cluster/gce/gci/configure-helper.sh

```diff
@@ -1858,6 +1858,17 @@ function prepare-etcd-manifest {
   fi
   # Replace the volume host path.
   sed -i -e "s@/mnt/master-pd/var/etcd@/mnt/disks/master-pd/var/etcd@g" "${temp_file}"
+  # Replace the runAsUser and runAsGroup
+  sed -i -e "s@{{runAsUser}}@${ETCD_RUNASUSER:-0}@g" "${temp_file}"
+  sed -i -e "s@{{runAsGroup}}@${ETCD_RUNASGROUP:-0}@g" "${temp_file}"
+  chown -R ${ETCD_RUNASUSER:-0}:${ETCD_RUNASGROUP:-0} /var/etcd/
+  # Replace capabilities
+  if [[ -n "${ETCD_RUNASUSER:-}" && -n "${ETCD_RUNASGROUP:-}" ]]; then
+    container_security_context=$(echo "\"securityContext\": {\"allowPrivilegeEscalation\": false, \"capabilities\": {\"drop\": [\"all\"]}},")
+  else
```

Comment:
else is not needed if you initialize container_security_context="". Also do we need the echo?

Reply:
Fixed.